projects/storefrontlib/src/cms-structure/page/component/cx-api.service.ts

```diff
@@ -43,7 +42,6 @@ export class CxApiService {
     // auth
     @Optional() public auth: AuthService,
     // cart
-    @Optional() public cart: CartService,
```

Comment:
I think we should place here `cart: ActiveCartService`

Reply:
Added